src/shared/esp-now-structs.h

```diff
@@ -47,6 +49,7 @@ struct EspNowDiscoveryResponseV1 {
 struct EspNowChallengeRequestV1 {
   uint8_t id = MESSAGE_ID_CHALLENGE_REQUEST_V1;
```

Comment:
As this is a breaking change (e.g. if nodes are updated they will not function until host has be updated, and vice versa), technically you should add this as a new version, e.g. `EspNowChallengeRequestV2`. This is why I have version to begin with :) (and then handle both versions in node and host impls). But, this might be overkill. If we just update all the nodes first (and can live with them non functioning until host is updated), then its OK.